app/src/main/sentences/cs/timer.dslf

```diff
@@ -0,0 +1,5 @@
+timer: high
+[set] časovač|(pípni za) .duration.;
+[set] nastav|vytvoř|zapni|pusť časovač (na? .duration. (s názvem .name.)?)|((s názvem) .name. na .duration.)
+[cancel] zastav|zruš|vypni|stop (.name.|(časovač (s názvem)? .name.))
+[query] kdy skončí|vyprší|(bude u konce) časovac? (s názvem)? .name.
```

Comment:
`;` is missing, I pushed a commit to fix it

Reply:
Fixed.